### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by kateryna3k**

### CODE REVIEW

**Review by Andriy-Makarenko — APPROVED** [review]
Good job!

File: app/players/dwarves/dwarf_blacksmith.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+class DwarfBlacksmith(Dwarf):
+    def __init__(
+            self, nickname: str, favourite_dish: str, skill_level: int
+    ) -> None:
+        super().__init__(nickname, favourite_dish)
+        self._skill_level = skill_level
+
+    def player_info(self) -> str:
+        return f"Dwarf blacksmith {self.nickname} " \
```
Comment (mltdll) [inline app/players/dwarves/dwarf_blacksmith.py:12]: I'm pretty sure we don't use `\` as line break here. Use parentheses instead:
```python
return (f"Dwarf blacksmith {self.nickname} "
        f"with skill of the {self._skill_level} level")
```

File: app/players/dwarves/dwarf_warrior.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+class DwarfWarrior(Dwarf):
+    def __init__(
+            self, nickname: str, favourite_dish: str, hummer_level: int
+    ) -> None:
+        super().__init__(nickname, favourite_dish)
+        self._hummer_level = hummer_level
+
+    def player_info(self) -> str:
+        return f"Dwarf warrior {self.nickname}. {self.nickname} " \
```
Comment (mltdll) [inline app/players/dwarves/dwarf_warrior.py:12]: Same here...

File: app/players/elves/druid.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+class Druid(Elf):
+    def __init__(
+        self, nickname: str, musical_instrument: str, favourite_spell: str
+    ) -> None:
+        super().__init__(nickname, musical_instrument)
+        self._favourite_spell = favourite_spell
+
+    def player_info(self) -> str:
+        return f"Druid {self.nickname}. " \
```
Comment (mltdll) [inline app/players/elves/druid.py:12]: And here...

File: app/players/elves/elf_ranger.py [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+class ElfRanger(Elf):
+    def __init__(
+            self, nickname: str, musical_instrument: str, bow_level: int
+    ) -> None:
+        super().__init__(nickname, musical_instrument)
+        self._bow_level = bow_level
+
+    def player_info(self) -> str:
+        return f"Elf ranger {self.nickname}. " \
```
Comment (mltdll) [inline app/players/elves/elf_ranger.py:12]: And here.

**Review by mltdll — CHANGES_REQUESTED** [review]
By the way, I see that someone already reviewed this. That's what happens when you don't tell us where you are in the table. :)

File: app/players/dwarves/dwarf_blacksmith.py (hunk 1 above)
Comment (kateryna3k) [reply app/players/dwarves/dwarf_blacksmith.py:12]: I'm not sure which is the best practice, so I want to leave it like it is

**Review by mltdll — APPROVED** [review]

File: app/players/dwarves/dwarf_blacksmith.py (hunk 1 above)
Comment (Y-Havryliv) [reply app/players/dwarves/dwarf_blacksmith.py:12]: I agree, using `( )` is better

**Review by Y-Havryliv — APPROVED** [review]
GJ!

**Pull request closed without merging**